cmd/thyme/main.go

```diff
@@ -3,36 +3,61 @@ package main
 import (
 	"encoding/json"
 	"fmt"
-	"log"
 	"os"
 	"runtime"
 
-	"github.com/jessevdk/go-flags"
+	"github.com/kovetskiy/godocs"
```

Comment:
`docopt/docopt.go` doesn't seem to be maintained. Specified fork at least solves some issues with parsing usage which original docopt can't handle and provide nifty pager support.